crates/dbsp/src/operator/sample.rs

```diff
@@ -76,6 +76,35 @@ where
         })
     }
 
+    /// Generates a uniform random sample of (key,value) pairs from `self`,
+    /// assuming that `self` contains exactly one value per key.
+    ///
+    /// Equivalent to `self.map(|(k, v)| (k, v)).stream_sample_keys()`,
+    /// but is more efficient.
+    #[allow(clippy::type_complexity)]
```

Comment:
We could also just make a type alias for `OrdZSet<(B::Key, B::Val), B::R>` and reuse it across the various functions
```rust
type ZSet<B> = OrdZSet<(B::Key, B::Val), B::R>
where
    B: BatchReader<Time = ()>;
```